waspc/data/Generator/templates/server/src/auth/providers/oauth/cookies.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  Request as ExpressRequest,
+  Response as ExpressResponse,
+} from "express";
+import { parseCookies } from "oslo/cookie";
+
+type CookieName = {
+  cookieName: string;
+};
```

Comment:
If you want a smart constructor, you can throw a symbol into the mix. That's gonna make it impossible for anyone to construct the value `CookieName` without using your functions. I can send a playground link if you don't know what I mean.

Although I'd prefer if we simplified it, as the other comment suggests.